### PR TITLE
OCLOMRS-513: Previewing a concept does not display a concept's mappings, answers and sets

### PR DESCRIPTION
**Pull request by Karuhanga**
# JIRA TICKET NAME:
[Concepts missing mappings, answers and sets on preview](https://issues.openmrs.org/browse/OCLOMRS-513)

# Summary:
When a user clicks 'preview concept', one of the items they expect to view are mappings, answers and sets.
Currently, mappings, answers and sets are not displayed

### CODE REVIEW

**Comment by Karuhanga** [comment]
Issue with drop in test coverage remedied by PR #404.

File: src/redux/actions/concepts/addBulkConcepts/index.js [hunk 1]
```diff
@@ -34,16 +34,16 @@ export const fetchFilteredConcepts = (source = 'CIEL', query = '', currentPage)
   const {
     bulkConcepts: { datatypeList, classList },
   } = getState();
-  let url = `orgs/${source}/sources/${source}/concepts/?${query}&limit=10&page=${currentPage}&verbose=true`;
+  let url = `orgs/${source}/sources/${source}/concepts/?${query}&limit=10&page=${currentPage}&verbose=true&includeMappings=1`;
 
-  if (datatypeList.length > 0 && classList.length > 0) {
-    url = `orgs/${source}/sources/${source}/concepts/?${query}&limit=10&page=${currentPage}&verbose=true&datatype=${datatypeList.join(',')}&conceptClass=${classList.join(',')}`;
+  // include dataType filter if it is present
```
Comment (dkayiwa) [inline src/redux/actions/concepts/addBulkConcepts/index.js]: What is the use of this comment?

File: src/components/bulkConcepts/component/MappingPreview.jsx [hunk 2]
```diff
@@ -23,10 +23,6 @@ const MappingPreview = ({ title, body }) => {
             Header: 'Relationship',
             accessor: 'map_type',
           },
-          {
```
Comment (dkayiwa) [inline src/components/bulkConcepts/component/MappingPreview.jsx]: How is this related to the mappings not being displayed?

Comment (Karuhanga) [reply src/components/bulkConcepts/component/MappingPreview.jsx]: It is not, directly. Given this functionality was not working before and this was a minor change raised within the same ticket, I felt it could be included.
New ticket?

File: src/redux/actions/concepts/addBulkConcepts/index.js (hunk 1 above)
Comment (Karuhanga) [reply src/redux/actions/concepts/addBulkConcepts/index.js]: Getting rid of this.

File: src/components/bulkConcepts/component/MappingPreview.jsx (hunk 2 above)
Comment (dkayiwa) [reply src/components/bulkConcepts/component/MappingPreview.jsx]: Yes that requires a new ticket.
